runtime/logging.go

```diff
@@ -192,3 +196,77 @@ func readBody(r io.ReadCloser) ([]byte, io.ReadCloser, error) {
 	}
 	return buf.Bytes(), ioutil.NopCloser(bytes.NewReader(buf.Bytes())), nil
 }
+
+// prettyFormatteer implements the Logrus Formatter interface
+// and provides a more simple, but easier to read, text formatter
+// option than the default logrus.TextFormatter.
+type prettyFormatter struct {
+}
+
+func isJSON(buf []byte) bool {
+	var tmp interface{}
+	err := json.Unmarshal(buf, &tmp)
+	return err == nil
```

Comment:
Nit: `return json.Unmarshal(buf, &tmp) == nil` but this is of course fine 😉

Reply:
I guess just a style preference. I like to use the `err` variable so at a glance it is easy to see what the boolean return is from, otherwise someone reading this (likely myself in a couple months) has to stop and remember/lookup what json.Unmarshal returns.